apple/internal/transition_support.bzl

```diff
@@ -148,7 +148,7 @@ def _cpu_string(*, environment_arch, platform_type, settings = {}):
             return cpu_value
         if cpu_value == "darwin_arm64":
             return "visionos_sim_arm64"
-        return "visionos_x86_64"
+        return "visionos_arm64"
```

Comment:
This is changing the default cpu value. From the release notes of Xcode 15.1 beta:
>Developing for visionOS requires a Mac with Apple silicon.